app/javascript/components/patient/Patient.js

```diff
@@ -172,7 +172,8 @@ class Patient extends React.Component {
                   this.props.details.black_or_african_american ? ' Black or African American' : ''
                 }${this.props.details.asian ? ' Asian' : ''}${this.props.details.american_indian_or_alaska_native ? ' American Indian or Alaska Native' : ''}${
                   this.props.details.native_hawaiian_or_other_pacific_islander ? ' Native Hawaiian or Other Pacific Islander' : ''
-                }`}</span>
+                }${this.props.details.race_unknown ? ' Unknown' : ''}${this.props.details.race_other ? ' Other' : ''}
```

Comment:
Small thing, but would it make sense to start separating these races with a comma or some other kind of delimiter? I just think "Asian Native Hawaiian or Other Pacific Islander Other" could be confusing to some people and they wouldn't know that it means "Asian AND Native Hawaiian or Other Pacific Islander AND Other". Maybe something we can ask the PO team?

Reply:
@timwongj it would make sense, but it's a slightly more complicated fix than it looks: I tried fixing it, but because you never know which race will be the first to display, it would require a real refactor rather than a one-line hotfix. That said, we probably won't have many tickets touching this specific bit of the code for a while so I'd be happy to add it in as soon as I have some spare time.

Reply:
sounds good!